### PR TITLE
[BULK UPDATE] DocuTune - Rebranding links

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort for the Microsoft Learn platform rebrand**

The Microsoft team is applying changes to content for the Microsoft Learn platform rebrand. Changes include modifications to current Microsoft Learn and Microsoft Docs terminology as well as associated domain name and content URL path changes.

Please review this draft PR and provide any necessary comments as soon as possible. Because these fixes are tightly scoped, we have approval to automatically merge these changes to the default branch for publication.

This PR was generated via DocuTune. It includes only targeted fixes and minor formatting adjustments.

CorrelationId: d8c4c3ac-d49a-4c95-9ddc-9822ca297907
InitiativeId: docutune-rebranding-2022-09
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: api/Excel.Workbook.AccuracyVersion.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The following table describes the possible settings for the **AccuracyVersion**
 > [!NOTE] 
 > Setting the **AccuracyVersion** property to a value other than 0, 1, or 2 will result in undefined behavior.
 
-<!--REMOVING THIS SECTION BECAUSE THERE IS NO TABLE; I FOUND ONE AT https://docs.microsoft.com/dotnet/api/microsoft.office.interop.excel._workbook.accuracyversion?view=excel-pia, BUT IT DOESN'T SEEM RIGHT).
+<!--REMOVING THIS SECTION BECAUSE THERE IS NO TABLE; I FOUND ONE AT https://learn.microsoft.com/dotnet/api/microsoft.office.interop.excel._workbook.accuracyversion?view=excel-pia, BUT IT DOESN'T SEEM RIGHT).
```
Comment (annmariehitchcock) [inline api/Excel.Workbook.AccuracyVersion.md:47]: Adding to broken link tab.

Comment (annmariehitchcock) [inline api/Excel.Workbook.AccuracyVersion.md:47]: Adding this to broken link tab.

**Comment by annmariehitchcock** [comment]
#docutune-ready

**Pull request merged**